DependencyInjection/DoctrineExtension.php

```diff
@@ -392,6 +392,10 @@ protected function ormLoad(array $config, ContainerBuilder $container)
             $container->removeDefinition('doctrine.orm.listeners.pdo_session_handler_schema_subscriber');
         }
 
+        if (! class_exists(LockStoreSchemaSubscriber::class)) {
```

Comment:
You are missing the import. Add a comment since which Symfony version is this available as well please.